src/style.css

```diff
@@ -0,0 +1,156 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* #region container */
+.item-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+/* #region image */
+.item-container__item-image {
+  margin: 32px 20px 41px;
+}
+
+.item-image {
+  width: 160px;
+  height: 134px;
+}

```

Comment:
```suggestion
  height: 134px;
  display: block;
  margin: 0 auto 40px;
}

```

Reply:
Be more attentive to my comments

![image](https://user-images.githubusercontent.com/63306833/125911582-8c5b0aae-7b20-48fd-afcf-c8b6d5327394.png)